.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
We should not be defining this file at the root of the monorepo. DO NOT check-in this file.